include/vast/Analyses/UninitializedValues.hpp

```diff
@@ -0,0 +1,522 @@
+// Copyright (c) 2024-present, Trail of Bits, Inc.
+
+#pragma once
+
+#include "vast/Interfaces/AST/DeclInterface.hpp"
+#include "vast/Interfaces/AST/TypeInterface.hpp"
+#include "vast/Interfaces/AST/StmtInterface.hpp"
+#include "vast/Interfaces/AST/ExprInterface.hpp"
+
+#include "vast/Analyses/CFG.hpp"
+#include "clang/AST/StmtVisitor.h"
+#include "llvm/ADT/BitVector.h"
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/PackedVector.h"
+#include "llvm/ADT/SmallVector.h"
+#include "vast/Analyses/FlowSensitive/DataflowWorklist.hpp"
+
+#include <optional>
+
+namespace vast::analyses {
+
+    // TODO
```

Comment:
instead of TODO comment use `VAST_UNIMPLEMENTED;` inside the body of the function

Reply:
and on all other places

Reply:
Done.